app/models/miq_worker.rb

```diff
@@ -49,7 +49,7 @@ class << self
   end
 
   def self.bundler_groups
-    %w[manageiq_default]
+    %w[manageiq_default ui_dependencies]
```

Comment:
This turns on ui for all non-workers which goes against the whole point of this, which was to avoid loading ui in non-ui stuff, since it's so expensive... cc @jrafanie

Reply:
yeah, I'm torn.  This will get it working until class/modules are moved back to core or extracted elsewhere.

I don't see any other approach though.  I'll make a github issue to decide where to move these module/classes.

Reply:
ok, opened https://github.com/ManageIQ/manageiq/issues/19863 to track moving the various module/classes